plugins/qtosgrave/qtosgviewer.cpp

```diff
@@ -2131,7 +2133,7 @@ void QtOSGViewer::_PostToGUIThread(const boost::function<void()>& fn, bool block
     GUIThreadFunctionPtr pfn(new GUIThreadFunction(fn, block));
     if( _listGUIFunctions.size() > 1000 ) {
```

Comment:
So that we can always push to `_listGUIFunctionsBuffer` without checking the size